test/games/planets_zebeth/exporter/test_planets_zebeth_game_exporter.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import annotations
+
+import multiprocessing
+from concurrent.futures import Future
+from contextlib import contextmanager
+from typing import TYPE_CHECKING
+from unittest.mock import ANY, MagicMock
+
+import planets_yapr.wrapper
```

Comment:
mypy is complaining. type ignore this and set a reminder/issue for you to add typing to the patcher.

Reply:
this? `import planets_yapr.wrapper #type: ignore[import-untyped]`

Reply:
Sounds right, can't check rn. Iirc am2r should have something similar.

Reply:
am2r doesn't have anything like that in the same test file for am2r